.github/workflows/ci.yaml

```diff
@@ -0,0 +1,118 @@
+---
+name: 'CI'
+
+on:
+  pull_request:
+    branches:
+      - '**'
+  push:
+    branches:
+      - 'main'
```

Comment:
ui tests are running only on release branch so do you need to push to release-* too ?

Reply:
It's ok for me to run UI tests once I create PR for the release branch, I don't want to waste resources on running UI tests every time I make a push.